src/Conference.Utils/Helpers/Settings.cs

```diff
@@ -91,33 +62,30 @@ string GetReminderId(string id)
 
         public string GetEventId(string id)
         {
-            return AppSettings.GetValueOrDefault(GetReminderId(id), string.Empty);
+            return Preferences.Get(GetReminderId(id), string.Empty);
         }
 
         public void RemoveReminderId(string id)
         {
-            AppSettings.Remove(GetReminderId(id));
+            Preferences.Set(GetReminderId(id), string.Empty);
         }
 
         public bool IsHackFinished(string id)
         {
-            return AppSettings.GetValueOrDefault("minihack_" + id, false);
+            return Preferences.Get("minihack_" + id, false);
         }
 
         public void FinishHack(string id)
         {
-            AppSettings.AddOrUpdateValue("minihack_" + id, true);
+            Preferences.Set("minihack_" + id, true);
         }
 
         const string LastFavoriteTimeKey = "last_favorite_time";
 
         public DateTime LastFavoriteTime
         {
-            get { return AppSettings.GetValueOrDefault(LastFavoriteTimeKey, DateTime.UtcNow); }
-            set
-            {
-                AppSettings.AddOrUpdateValue(LastFavoriteTimeKey, value);
-            }
+            get { return new DateTime(Preferences.Get(LastFavoriteTimeKey, DateTime.UtcNow.Ticks)); }
+            set { Preferences.Set(LastFavoriteTimeKey, value.Ticks); }
```

Comment:
@jamesmontemagno I believe that @mattleibow added `DateTime` support, can you use it here?